client/state/themes/reducer.js

```diff
@@ -162,6 +165,34 @@ export function themeRequests( state = {}, action ) {
 }
 
 /**
+ * Returns the updated Jetpack site wpcom theme install requests state after an action has been
+ * dispatched. The state reflects a mapping of site ID, theme ID pairing to a
+ * boolean reflecting whether a request for the theme install is in progress.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export function installThemeOnJetpackRequests( state = {}, action ) {
```

Comment:
I can change it to have word progress - I have no preferences here :) I made it this way to follow general pattern of naming reducers we have in this file:
`themeRequests` , `activeThemeRequests` and couple of others indicate if the "process"  is ongoing but don't have word process in theme.